src/css/_docs.scss

```diff
@@ -229,7 +238,7 @@
     }
     &-supported {
       display: inline-block;
-      margin: 0;
+      margin: -6px 0 5px;
```

Comment:
I can change it to position: relative / bottom 6 if you prefer? It does work but happy to change it